src/test/java/org/apache/commons/io/IOUtilsTestCase.java

```diff
@@ -118,7 +116,7 @@ public void setUp() {
             if (!testFile.getParentFile().exists()) {
                 throw new IOException("Cannot create file " + testFile + " as the parent directory does not exist");
             }
-            final BufferedOutputStream output = new BufferedOutputStream(new FileOutputStream(testFile));
+            final BufferedOutputStream output = new BufferedOutputStream(Files.newOutputStream(testFile.toPath()));
```

Comment:
In these test methods, you keep calling `toPath()` over and over, why not just call it once?

Reply:
HI @garydgregory 
you're right. Changed.
TY